internal/commands/app/create_inputs.go

```diff
@@ -217,3 +239,15 @@ func (i createInputs) args(omitDryRun bool) []flags.Arg {
 	}
 	return args
 }
+
+func getDefaultPath(wd string, localPath string) string {
+	i := 1
+	for {
```

Comment:
a couple of thoughts here:
  - should assign `localPath + "-" + strconv.Itoa(i)` to a var within this loop
  - the error check seems off here, doesn't it want to be `if err == nil && !found`?
  - it would be nice if we had a failsafe here (maybe append a `primitive.NewObjectID().Hex()` if we exhaust file options `1-9`?)
  - we should add some unit tests for this as well